content/en/developers/dogstatsd/dogstatsd_mapper.md

```diff
@@ -0,0 +1,129 @@
+| `<PROFILE_PREFIX>`      | The metric name prefix associated to this profile.                                                                                        | yes                     |
+| `<METRIC_TO_MATCH>`     | The metric name to extract groups from with the [Wildcard](#wildcard-match-pattern) or [Regex](#regex-match-pattern) match logic.         | yes                     |
+| `<MATCH_TYPE>`          | The type of match to apply to the `<METRIC_TO_MATCH>`. Either [`wildcard`](#wildcard-match-pattern) or [`regex`](#regex-match-pattern)    | no, default: `wildcard` |
+| `<MAPPED_METRIC_NAME>`  | The new metric name to send to Datadog with the tags defined in the same group.                                                           | yes                     |
+| `<TAG_KEY>`             | The Tag key to associate to the tags collected.                                                                                           | no                      |
+| `<TAG_VALUE_TO_EXPAND>` | The tags collected from the `<MATCH_TYPE>` to inline.                                                                                     | no                      |
+
+## Wildcard Match Pattern
+
+The wildcard match pattern matches dot-separated metric names using `*` as wildcard. The metric name must be only composed of alphanumeric, `.`, and `_` characters for this pattern to work.
+Groups extracted can then be expanded with the `$n` format e.g. `$1`, `$2`, `$3`... or the `${n}` format e.g. `${1}`, `${2}`, `${3}`, ...
+
+For instance, if you have the metric `custom_metric.value_1.value_2` with the following mapping group configuration:
+
+```yaml
+dogstatsd_mapper_profiles:
+    - name: my_custom_metric_profile
+      prefix: custom_metric
+      mappings:
+          - match: 'custom_metric.*'
+            match_type: wildcard
+            name: custom_metric
+            tags:
+                tag_key_1: '$1'
+                tag_key_2: '$2'
+```
+
+It would send the metric `custom_metric` to Datadog with the tags `tag_key_1:value_1` and `tag_key_2:value_2`.
+
+**Note**: If the incoming metric have already at least one tag, no mapping is applied.
+
+## Regex Match Pattern
+
+The regex match pattern matches metric names using regex patterns. Compared to the wildcard match pattern, it allows to define captured groups that contain `.`.
+Groups extracted can then be expanded with the `$n` format e.g. `$1`, `$2`, `$3`... or the `${n}` format e.g. `${1}`, `${2}`, `${3}`, ...
+
+For instance, if you have the metric `custom_metric.value_1.value.with.dots._2` with the following mapping group configuration:
+
+```yaml
+dogstatsd_mapper_profiles:
+    - name: my_custom_metric_profile
+      prefix: custom_metric
+      mappings:
+          - match: 'custom_metric\.([\w_]+)\.(.+)'
```

Comment:
Same here, I think using something with more than just the prefix is better e.g. `custom_metric.process`  (might need update of other occurrences too)